packages/react-admin/src/reducer/i18n/index.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers } from 'redux';
+import locale from './locale';
+import messages from './messages';
+import loading from './loading';
+import requested from './requested';
+
+export default initialLocale => {
+    return combineReducers({
+        locale: locale(initialLocale),
+        messages,
+        loading,
+        requested,
```

Comment:
I don't understand this `requested` reducer. Where is it used ?

Reply:
It's not used in `react-admin`, so maybe it shouldn't be there. It's there so an user can query the locale being loaded. `{locale:'en',requested:null}` -> `{type:CHANGE_LOCALE,payload:'nl'}` -> `{locale:'en',requested:'nl'}` ->possibility to show ui loading `nl` -> `{type:CHANGE_LOCALE_SUCCESS}` -> `{locale:'nl',requested:null}`

Reply:
I think this should stay in userland then